infrastructure/code/code_html.go

```diff
@@ -46,6 +47,42 @@ func getDataFlowHeadingHtml(issue snyk.CodeIssueData) string {
 	return fmt.Sprintf("Data Flow - %d %s", dataFlowCount, stepWord)
 }
 
+func getIgnoreDetailsHtml(isIgnored bool, ignoreDetails *snyk.IgnoreDetails) (ignoreDetailsHtml string, visibilityClass string) {
+	if !isIgnored {
+		return "", "hidden"
+	}
+
+	labels := []string{"Category", "Ignored On", "Expiration", "Ignored By"}
+
+	detailsMap := map[string]string{
+		"Category":   ignoreDetails.Category,
+		"Ignored On": formatDate(ignoreDetails.IgnoredOn),
+		"Expiration": ignoreDetails.Expiration,
+		"Ignored By": ignoreDetails.IgnoredBy,
+	}
+
+	pairs := `<div class="ignore-details-wrapper">`
+	for _, label := range labels {
```

Comment:
This is tidy, but hard to understand. Maybe extract the for-loop to make it easier to read? Love the concise code though.